src/App/utils/resourcesByType/index.js

```diff
@@ -886,10 +890,16 @@ export const resourcesByType = {
 
       login: {
         useCase: `Uses local storage "token" if it exists. Adds/updates local storage "token" if there is a "jwt" query param in the URL from an egghead login URL redirect.`,
+        createExamples: () => [
+          login(),
+        ],
       },
 
       logout: {
         useCase: `Removes "token" from local storage and refreshes the page to log the user out.`,
+        createExamples: () => [
+          logout(),
+        ],
```

Comment:
This is very meta, when you go to this resource "example" it will actually log you out 😂